sql/data_model.sql

```diff
@@ -0,0 +1,64 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
```

Comment:
I've been mulling this over a bit. I think it will almost definitely make sense to keep a table like this around, but it's not strictly necessary for the primitive endpoints and could potentially be thought of as an analytics table.

I think for phone primitives the only thing we really need is an index of forms to cases that were touched (so that we can rebuild cases if necessary), as well as some reliable persistent storage of the raw form contents.

(not sure this type of thinking is useful, just pointing out)